### PR TITLE
docs: clarify descriptions for vips_extract_band docs

### PR DESCRIPTION
**Pull request by knarewski**
Hi, thank you for this great project and amazing maintenance efforts!

When playing with `libvips` it wasn't obvious to me how to use `vips_extract_band` despite reading the docs, so I'm providing suggestion on making the relevant parts slightly clearer. I tried to guess the right places with code search, please let me know if I misplaced my changes!

### CODE REVIEW

File: cplusplus/include/vips/VImage8.h [hunk 1]
```diff
@@ -2994,7 +2994,7 @@ class VImage : public VObject {
 	 * **Optional parameters**
 	 *   - **n** -- Number of bands to extract, int.
 	 *
-	 * @param band Band to extract.
+	 * @param band Index of a first band to extract.
```
Comment (jcupitt) [inline cplusplus/include/vips/VImage8.h:2997]: This file is generated automatically, so you don't need to change this yourself.

File: libvips/conversion/extract.c [hunk 2]
```diff
@@ -459,7 +459,7 @@ vips_extract_band_init(VipsExtractBand *extract)
  * vips_extract_band: (method)
  * @in: input image
  * @out: (out): output image
- * @band: band to extract
+ * @band: index of a first band to extract
```
Comment (jcupitt) [inline libvips/conversion/extract.c:462]: nitpick: I'd just write "index of first band to extract".

**Review by jcupitt — COMMENTED** [review]
Thanks for the doc polishing. I made a couple of small comments.

**Comment by knarewski** [comment]
Thank you for a blazingly fast review, I've applied your suggestions :)

**Comment by jcupitt** [comment]
No problem, thanks for the PR!

**Pull request merged**